### ISSUE TITLE
store sshomp actor id in country table

### ISSUE THREAD

**Issue by stefanprobst** [DARIAH-ERIC/dariah-unr#51]
in order to synchronize services from the ssh open marketplace, it is not enough to [query for services with keyword "DARIAH Resource"](https://marketplace.sshopencloud.eu/search?f.keyword=DARIAH+Resource), we also need to map these services to countries in our database. since countries are not represented in the sshomp data model, we need to map actor names like "DARIAH-AT" to a country.

we should save the sshomp actor id as a field in our country table (maybe also the name).

also, services should also store the sshomp id/url - we have a `marketplaceUrl` field for software, but not for services.

---

on the sshomp side, we may want to add an explicit DARIAH `identifierService`. and, we should include `accessibleAt` in search results from `/api/item-search` so we can populate the service's `url` field.

---

still think syncing the other way around (pushing services created in unr to sshomp via sshomp api) makes more sense. e.g. how would we populate the "service size" field in unr when ingesting services from sshomp.

---

also, let's make `marketplaceStatus` a proper enum (probably reflecting the actual status field in sshomp), instead of yes/no.

---

do we have a way to distinguish Tools (Software) and Service in MP? (I believe we have a dedicated type-field)

**Comment by laureD19** [comment]
see below re. first point. Should I go for baserow addition instead? Or better to stop updating baserow?
some countries do not have a representation yet in the Marketplace. 

country | actor ID in the SSH Open Marketplace | Name | actor-search
-- | -- | -- | --
Austria | 9403 | CLARIAH-AT | https://marketplace-api.sshopencloud.eu/api/actor-search?q=CLARIAH-AT
Belgium | 3235 | DARIAH-BE | https://marketplace-api.sshopencloud.eu/api/actor-search?q=DARIAH-BE
Bosnia and Herzegovina |   |   |  
Bulgaria | 3754 | CLaDA-BG | https://marketplace-api.sshopencloud.eu/api/actor-search?q=CLADA-BG
Croatia | 3403 | DARIAH-HR | https://marketplace-api.sshopencloud.eu/api/actor-search?q=DARIAH-HR
Cyprus |   |   |  
Czech Republic | 3804 | LINDAT/CLARIAH-CZ | https://marketplace-api.sshopencloud.eu/api/actor-search?q=LINDAT/CLARIAH-CZ
Denmark | 9560 | DARIAH-DK | https://marketplace-api.sshopencloud.eu/api/actor-search?q=DARIAH-DK
France | 10860 | DARIAH-FR | https://marketplace-api.sshopencloud.eu/api/actor-search?q=DARIAH-FR
Germany | 2868 | DARIAH-DE | https://marketplace-api.sshopencloud.eu/api/actor-search?q=DARIAH-DE
Greece | 3502 | DARIAH-GR / DYAS | https://marketplace-api.sshopencloud.eu/api/actor-search?q=DARIAH-GR
Ireland | 3755 | DARIAH-IE | https://marketplace-api.sshopencloud.eu/api/actor-search?q=DARIAH-IE
Italy | 10226 | DARIAH-IT | https://marketplace-api.sshopencloud.eu/api/actor-search?q=DARIAH-IT
Luxembourg | 9561 | DARIAH-LU | https://marketplace-api.sshopencloud.eu/api/actor-search?q=DARIAH-LU
Malta |   |   |  
Netherlands | 9562 | CLARIAH-NL | https://marketplace-api.sshopencloud.eu/api/actor-search?q=CLARIAH-NL
Poland | 3752 | DARIAH-PL | https://marketplace-api.sshopencloud.eu/api/actor-search?q=DARIAH-PL
Portugal | 3553 | DARIAH-PT / ROSSIO | https://marketplace-api.sshopencloud.eu/api/actor-search?q=DARIAH-PT
Serbia |   |   |  
Slovenia | 9133 | DARIAH-SI | https://marketplace-api.sshopencloud.eu/api/actor-search?q=DARIAH-SI
Spain |   |   |  
Switzerland | 10002 | DARIAH-CH | https://marketplace-api.sshopencloud.eu/api/actor-search?q=DARIAH-CH

**Comment by stefanprobst** [comment]
thanks! no need to update anything in baserow!

**Comment by stefanprobst** [comment]
are dariah member country actors always listed with role "reviewer"?

**Comment by stefanprobst** [comment]
one tool/service tagged with "DARIAH resource" is not connected with a specific country: https://marketplace.sshopencloud.eu/tool-or-service/VZRMmh

**Comment by laureD19** [comment]
> are dariah member country actors always listed with role "reviewer"?

yes. This is included in the guidelines we ask them to follow, and an element checked by moderators of the dariah resources

**Comment by laureD19** [comment]
> are dariah member country actors always listed with role "reviewer"?

yes

**Comment by laureD19** [comment]
> one tool/service tagged with "DARIAH resource" is not connected with a specific country: https://marketplace.sshopencloud.eu/tool-or-service/VZRMmh

this can indeed happen. In that specific case, it is a tool developed by a dariah-eu employee, but we are waiting for more cases, from the dariah working groups for example. Resources part of the dariah network but not affiliated to any country specifically.

**Comment by laureD19** [comment]
regarding the last question in the initial comment on how to distinguish between services and software. All software should be tagged with the "Software" value in the `resource-category` property field. See for example: 
https://marketplace-api.sshopencloud.eu/api/item-search?d.resource-category=%22Software%22&f.keyword=DARIAH%20Resource 

There will probably be inconsistencies with what we have in the portfolio/database. And I was counting on these inconsistencies to actually improve the curation/tagging of the software in the marketplace.

I hope it helps a bit!
